draft-bbs-signatures.md

```diff
@@ -346,18 +346,27 @@ Procedure:
 
 Sign computes a signature from SK, PK, over a vector of messages. This method
 describes deterministic signing. For threshold signing, XOF can be replaced
-with a PRF due to the insecurity of deterministic threshold signing.
+with a PRF due to the insecurity of deterministic threshold signing. 
+
+The method takes as an input a list of the generators (H_s, H_d, H_1, ..., H_L). The first generator from that list (H_s), is used for the blinding value (s) of the signature. The second generator (H_d), is used to sign the signature's domain separation tag (sig_dst), which binds both signature and proof to a specific domain. Finally, the rest of the generators (H_1, ..., H_L), are used for the signed messages.
 
 ```
-signature = Sign(SK, PK, (msg_1,..., msg_L), (H_1,..., H_L))
+signature = Sign(SK, PK, (msg_1,..., msg_L), (H_s, H_d, H_1, ..., H_L))
```

Comment:
Im still pondering whether we make H_s and H_d cipher suite defined values rather than picking two generators and setting these as global fixed generators for all cipher suites, this would reduce this API's complexity by not having to supply them.

Reply:
Agreed. Updated the PR to define two more generator seeds for `H_s` and `H_d`. I also prefer this way. Since those generators are no longer calculated exactly the same as the message generators we may not need to pass them as input, as it was raised to the WG call.